### DISCUSSION TITLE
All files open in Browse mode only

### DISCUSSION THREAD

**Discussion by alcampin** [codefori/vscode-ibmi#1605] [Q&A]
I am attempting in VSCODE to open a source member in an IBM i source file. The file QRPGLESRC is marked for *PUBLIC *ALL authority but when I attempt to edit the file it says read only. It keeps insisting that the file is read only. Anyone else run into this?

**Comment by worksofliam** [comment]
Hey @alcampin 

I would first recommend you check that the connection is not 'protected'. You can do this in the connection settings:

<img width="1362" alt="Screenshot 2023-10-20 at 2 08 07 PM" src="https://github.com/codefori/vscode-ibmi/assets/3708366/33a4a706-c92f-4136-80f5-457972d3c05c">

Second, if there's no luck there please share the contents of your Code for IBM i output channel after you have opened the member in browse mode.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">

**Comment by alcampin** [comment]
[output.txt](https://github.com/codefori/vscode-ibmi/files/13058170/output.txt)

**Reply by alcampin** [reply]
Connection is not in Read only

**Reply by worksofliam** [reply]
@alcampin is the User Library List view, what is your current library set to? It looks like `*NONE` from the logs. I would try changing it to anything else - even `QGPL`
